sidekick_core/lib/src/commands/update_command.dart

```diff
@@ -38,47 +73,157 @@ class UpdateCommand extends Command {
             ) ??
             Version.none;
 
-    if (versionToInstall <= currentSidekickCliVersion) {
+    final DartPackageBundle packageToInstall;
+    if (version != null) {
+      if (!version.isPreRelease &&
```

Comment:
I was confused by the condition because it took me some time to understand that `availableVersions` never contains pre-release versions 
```suggestion
      // `availableVersions` only contains non-pre-release versions. If a pre-release version is explicitly given, install it nonetheless 
      if (!version.isPreRelease &&
```